tests/aio/query/test_query_session_pool.py

```diff
@@ -55,6 +59,36 @@ async def callee(session: QuerySession):
         with pytest.raises(CustomException):
             await pool.retry_operation_async(callee)
 
+    @pytest.mark.parametrize(
+        "tx_mode",
+        [
+            (None),
+            (ydb.QuerySerializableReadWrite()),
+            (ydb.QuerySnapshotReadOnly()),
+            (ydb.QueryOnlineReadOnly()),
+            (ydb.QueryStaleReadOnly()),
+        ],
+    )
+    @pytest.mark.asyncio
+    async def test_retry_tx_normal(self, pool: QuerySessionPool, tx_mode: Optional[ydb.BaseQueryTxMode]):
```

Comment:
It need tests for really retry retriable errors. Simplest the error for reproduce- TLI.